src/DependencyInjection/SonataAdminExtension.php

```diff
@@ -115,12 +115,18 @@ public function load(array $configs, ContainerBuilder $container)
         $container->setParameter('sonata.admin.configuration.default_icon', $config['options']['default_icon']);
         $container->setParameter('sonata.admin.configuration.breadcrumbs', $config['breadcrumbs']);
 
+        // NEXT_MAJOR: Remove this block and uncomment the one below.
         if (null === $config['security']['acl_user_manager'] && isset($bundles['FOSUserBundle'])) {
             $container->setParameter('sonata.admin.security.acl_user_manager', 'fos_user.user_manager');
         } else {
             $container->setParameter('sonata.admin.security.acl_user_manager', $config['security']['acl_user_manager']);
         }
 
+        // NEXT_MAJOR: Uncomment this code.
+        //if (null !== $config['security']['acl_user_manager']) {
+        //    $container->setAlias('sonata.admin.security.acl_user_manager', $config['security']['acl_user_manager']);
```

Comment:
take a look at my PR to see the correct alias, otherwise it does not work (at least on my tests)

Reply:
I also left this one because from the controller we are fetching it with `get('sonata.admin.security.acl_user_manager')`